CMakeLists.txt

```diff
@@ -276,7 +285,7 @@ endif()
 #
 find_program(
   CLANG_FORMAT_EXECUTABLE
-  NAMES clang-format clang-format-3.9 clang-format-3.8
+  NAMES clang-format-3.8
```

Comment:
> Why did you remove support for other versions of clang-format?

Well, I observed that they format the code slightly different (even between 3.8 and 3.9) with the same configuration. So I'd like to we use one specific version, then figure out how can we handle this later. I chose ClangFormat 3.8 because it is the latest version supported by trusty.



> Could we use clang-format-${LLVM_VERSION} to make this slightly more portable?

That's an good idea. But still, we should stay with ClangFormat 3.8 until we figure out how to get the same formatted code across different versions. Let me create an issue to track this.

> Even better, does Clang install a Config.cmake file that includes the path to this executable?

[Hm, I don't think so](http://packages.ubuntu.com/trusty-updates/amd64/clang-format-3.8/filelist).